x-pack/plugins/cloud_security_posture/public/pages/rules/index.tsx

```diff
@@ -41,6 +45,23 @@ const getRulesBreadcrumbs = (
   return breadCrumbs;
 };
 
+const getRulesSharedValues = (packageInfo: PackagePolicy, agentInfo: AgentPolicy) => {
+  const enabledPackage = packageInfo.inputs.find((input) => input.enabled);
+  const matchingKubeDeployOption = kubeDeployOptions.find(
+    (option) => option.value === enabledPackage?.type
+  );
+
+  // TODO: most of the values here should come from the integration directly
+  return [
+    { title: 'Integration', description: 'KSPM' },
+    !!matchingKubeDeployOption?.label && {
+      title: 'Type',
+      description: matchingKubeDeployOption?.label,
+    },
+    { title: 'Agent', description: agentInfo.name },
```

Comment:
`Integration` and `Type` should both come from the integration it self, along with the `Benchmark` which is too hacky to mess with since we need this change from the integration anyway.
 
I'll open a task and link it here